pyglmnet/datasets.py

```diff
@@ -119,7 +159,7 @@ def fetch_community_crime_data():
     return X, y
 
 
-def fetch_rgc_data(dpath, accept_rgcs_license=False):
+def fetch_rgc_data(dpath=None, accept_rgcs_license=False):
```

Comment:
I had forgotten. Can you update the `api.rst` page with the datasets?

also you can add a line in `whats_new.rst` for adding the new function (not example)